src/sagemaker/jumpstart/exceptions.py

```diff
@@ -150,6 +150,7 @@ def __init__(
                 model. (Default: None).
 
         """
+        version = version or "*"
```

Comment:
sanity check: this would change the behavior on line 157 right? Is that intentional?

Reply:
This was intentional since `version` was being supplied as `None`, not being omitted. 
I'm going to try removing this tho